File: Clase Punto.py
class Punto:
    def __init__(self, x: float, y: float):
        self.x = x
        self.y = y

    def coord_cartesianas(self):
        return (self.x, self.y)

    def coord_polares(self):
        r = (self.x**2 + self.y**2) ** 0.5
        theta = math.atan2(self.y, self.x)
        return (r, theta)

    def __str__(self):
        return f"Punto({self.x}, {self.y})"

File: Clase Punto.java
public class Punto {
    private float x;
    private float y;

    public Punto(float x, float y) {
        this.x = x;
        this.y = y;
    }

    public float[] coordCartesianas() {
        return new float[]{x, y};
    }

    public float[] coordPolares() {
        float r = (float) Math.sqrt(x * x + y * y);
        float theta = (float) Math.atan2(y, x);
        return new float[]{r, theta};
    }

    @Override
    public String toString() {
        return "Punto(" + x + ", " + y + ")";
    }
}
